modules/apps/transfer/types/packet_test.go

```diff
@@ -719,20 +722,57 @@ func TestUnmarshalPacketData(t *testing.T) {
 	var (
 		packetDataBz []byte
 		version      string
+		encoding     string
 	)
 
 	testCases := []struct {
 		name     string
 		malleate func()
-		expError error
+		expError string
```

Comment:
wdyt of using instead an `error` and `RequireErrorIsOrContains` (https://github.com/cosmos/ibc-go/blob/main/testing/utils.go#L88) and either:

- Use the exact error
- Use errors.New() 

you can see an example hre: https://github.com/cosmos/ibc-go/blob/main/modules/apps/29-fee/types/msgs_test.go#L22-L94

Reply:
I like that! I will try it out :)